packages/@aws-cdk/aws-cloudwatch/lib/metric.ts

```diff
@@ -446,6 +447,23 @@ export class Metric implements IMetric {
     });
   }
 
+  /**
+   * Make a new Anomaly Detection Alarm for this metric.
+   */
+  public createAnomalyDetectionAlarm(scope: Construct, id: string, props: CreateAnomalyDetectionAlarmOptions): AnomalyDetectionAlarm {
```

Comment:
same comments here from the other `createAnomalyDetectionAlarm

Reply:
Done